recipes/tandem-genotypes/meta.yaml

```diff
@@ -0,0 +1,29 @@
+{% set version = "1.2.0" %}
+
+package:
+  name: tandem-genotypes
+  version: {{ version }}
+
+source:
+  url: https://github.com/mcfrith/tandem-genotypes/archive/{{ version }}.tar.gz
+  sha256: a5a48ca33a2f11bdee4fcfa8dc2da4306b958b141b40014c1a0362a7bdd836bd
+
+build:
```

Comment:
add `noarch: python`
https://bioconda.github.io/linting.html#should-be-noarch

Reply:
Many thanks for looking at this.

But your link says: "Don’t do this if your Python package has a command line interface", which it does.

Also this:
https://bioconda.github.io/guidelines.html#id3

I'm confused...

Reply:
@mcfrith the documentation is lagging behind a recent update of the bioconda recipes and build system to conda3.